2019-02-25 12:09:17,365 DEBUG    root            Loaded Command Group: [u'gcloud', u'compute']
2019-02-25 12:09:17,385 DEBUG    root            Loaded Command Group: [u'gcloud', u'compute', u'ssh']
2019-02-25 12:09:17,388 DEBUG    root            Running [gcloud.compute.ssh] with arguments: [--zone: "us-east1-b", [USER@]INSTANCE: "tensorboard"]
2019-02-25 12:09:19,604 DEBUG    root            (gcloud.compute.ssh) PERMISSION_DENIED: The caller does not have permission
Traceback (most recent call last):
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 987, in Execute
    resources = calliope_command.Run(cli=self, args=args)
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 795, in Run
    resources = command_instance.Run(args)
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/surface/compute/ssh.py", line 161, in Run
    instance = ssh_helper.GetInstance(client, instance_ref)
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/googlecloudsdk/command_lib/compute/ssh_utils.py", line 391, in GetInstance
    return client.MakeRequests([request])[0]
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/googlecloudsdk/api_lib/compute/client_adapter.py", line 91, in MakeRequests
    progress_tracker=progress_tracker,
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/googlecloudsdk/api_lib/compute/request_helper.py", line 253, in MakeRequests
    requests=requests, http=http, batch_url=batch_url)
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/googlecloudsdk/api_lib/compute/batch_helper.py", line 106, in MakeRequests
    batch_request_callback=batch_checker.BatchCheck)
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/third_party/apitools/base/py/batch.py", line 222, in Execute
    batch_http_request.Execute(http)
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/third_party/apitools/base/py/batch.py", line 494, in Execute
    self.__callback(response, exception)
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/googlecloudsdk/api_lib/compute/batch_helper.py", line 73, in BatchCheck
    is_batch_request=True)
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/googlecloudsdk/api_lib/util/apis.py", line 233, in PromptToEnableApi
    enable_api.EnableServiceIfDisabled(project, service_token)
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/googlecloudsdk/api_lib/services/enable_api.py", line 148, in EnableServiceIfDisabled
    if IsServiceEnabled(project_id, service_name):
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/googlecloudsdk/api_lib/services/enable_api.py", line 100, in IsServiceEnabled
    exceptions.ReraiseError(e, exceptions.ListServicesPermissionDeniedException)
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/googlecloudsdk/api_lib/services/exceptions.py", line 55, in ReraiseError
    core_exceptions.reraise(klass(api_lib_exceptions.HttpException(err)))
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/googlecloudsdk/core/exceptions.py", line 139, in reraise
    six.reraise(type(exc_value), exc_value, tb)
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/googlecloudsdk/api_lib/services/enable_api.py", line 92, in IsServiceEnabled
    field='services'):
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/third_party/apitools/base/py/list_pager.py", line 77, in YieldFromList
    global_params=global_params)
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/googlecloudsdk/third_party/apis/servicemanagement/v1/servicemanagement_v1_client.py", line 949, in List
    config, request, global_params=global_params)
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 731, in _RunMethod
    return self.ProcessHttpResponse(method_config, http_response, request)
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 737, in ProcessHttpResponse
    self.__ProcessHttpResponse(method_config, http_response, request))
  File "/Users/ionlights/Downloads/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 604, in __ProcessHttpResponse
    http_response, method_config=method_config, request=request)
ListServicesPermissionDeniedException: PERMISSION_DENIED: The caller does not have permission
2019-02-25 12:09:19,608 ERROR    root            (gcloud.compute.ssh) PERMISSION_DENIED: The caller does not have permission
